packages/bot-web-ui/src/utils/help-content/__tests__/help-content.config.spec.ts

```diff
@@ -0,0 +1,702 @@
+        });
+
+        it('should have the correct types and example_id', () => {
+            expect(config.math_constrain[0]).toEqual({ type: TEXT });
+            expect(config.math_constrain[1]).toEqual({ type: BLOCK });
+            expect(config.math_constrain[2]).toEqual({ type: TEXT });
+            expect(config.math_constrain[3]).toEqual({ type: TEXT });
+            expect(config.math_constrain[4]).toEqual({ type: EXAMPLE, example_id: 'constrain' });
+        });
+    });
+
+    describe('text_print', () => {
+        it('should have 3 items', () => {
+            expect(config.text_print).toHaveLength(3);
+        });
+
+        it('should have the correct types', () => {
+            expect(config.text_print[0]).toEqual({ type: TEXT });
+            expect(config.text_print[1]).toEqual({ type: BLOCK });
+            expect(config.text_print[2]).toEqual({ type: TEXT });
+        });
+    });
+
+    describe('text_prompt_ext', () => {
+        it('should have 3 items', () => {
+            expect(config.text_prompt_ext).toHaveLength(3);
+        });
+
+        it('should have the correct types', () => {
+            expect(config.text_prompt_ext[0]).toEqual({ type: TEXT });
+            expect(config.text_prompt_ext[1]).toEqual({ type: BLOCK });
+            expect(config.text_prompt_ext[2]).toEqual({ type: TEXT });
+        });
+    });
+
+    describe('tick_analysis', () => {
+        it('should have 2 items', () => {
+            expect(config.tick_analysis).toHaveLength(2);
+        });
+
+        it('should have the correct types', () => {
+            expect(config.tick_analysis[0]).toEqual({ type: TEXT });
+            expect(config.tick_analysis[1]).toEqual({ type: BLOCK });
+        });
+    });
+
+    describe('read_details', () => {
+        it('should have 14 items', () => {
+            expect(config.read_details).toHaveLength(14);
+        });
+
+        it('should have the correct types', () => {
+            expect(config.read_details[0]).toEqual({ type: TEXT });
+            expect(config.read_details[1]).toEqual({ type: BLOCK });
+            expect(config.read_details[2]).toEqual({ type: TEXT });
+            expect(config.read_details[3]).toEqual({ type: TEXT });
+            expect(config.read_details[4]).toEqual({ type: TEXT });
+            expect(config.read_details[5]).toEqual({ type: TEXT });
+            expect(config.read_details[6]).toEqual({ type: TEXT });
+            expect(config.read_details[7]).toEqual({ type: TEXT });
+            expect(config.read_details[8]).toEqual({ type: TEXT });
+            expect(config.read_details[9]).toEqual({ type: TEXT });
+            expect(config.read_details[10]).toEqual({ type: TEXT });
+            expect(config.read_details[11]).toEqual({ type: TEXT });
+            expect(config.read_details[12]).toEqual({ type: TEXT });
+            expect(config.read_details[13]).toEqual({ type: TEXT });
+        });
+    });
```

Comment:
I think we should follow the above suggestion on any cases that have more than 5 expect @maryia-matskevich-deriv

Reply:
the file has been removed